### DISCUSSION TITLE
[Feature Request]: Respect XDG

### DISCUSSION THREAD

**Discussion by gldtn** [beyondcode/herd-community#359] [Ideas]
### Feature Description

Currently everytime I update PHP, Herd creates a new .zshrc file inside my $HOME folder which is a bit annoying because I have $ZDOTDIR set to $HOME/.config/zsh

It would be nice to have Herd respect the desktop environment variables. I've looked in Herd docs and did not find any mentions of this, so forgive me if it's already somehow implemented.

**Comment by eleftrik** [comment]
I agree with @gldtn. IMO Herd should not update .zshrc.

**Comment by JeffreyDavidson** [comment]
Is there an update on this feature request?

**Comment by svenluijten** [comment]
This is (unfortunately) still an issue. The supposed fix is to utilize `$ZDOTDIR/.zshrc` instead of hardcoding `~/.zshrc` to write the config to (or another, more sustainable solution entirely).

FWIW this is also what caused [this issue](https://github.com/beyondcode/herd-community/issues/787#issuecomment-4320314382) for me.
